src/models/comment.js

```diff
@@ -4,9 +4,9 @@ const mongoose = require("mongoose");
 //Creates comment schema
 
 const commentSchema = new mongoose.Schema({
-  userID: { type: Schema.Types.ObjectId, ref: "User" },
+  user: { type: mongoose.Schema.Types.ObjectId, ref: User },
```

Comment:
I believe you still need quotes around User, `ref: "User"`

Reply:
For some reason the quotes don't work on my end. It works without the quotes though. Not sure why.